pay-api/src/pay_api/services/payment_account.py

```diff
@@ -818,13 +818,14 @@ def publish_account_mailer_event_on_creation(self):
     def _publish_queue_message(self, payload: dict, message_type: str):
         """Publish to account mailer to send out confirmation email or notification email."""
         try:
-            gcp_queue_publisher.publish_to_queue(
+            gcp_queue_message.publish_to_queue(
                 QueueMessage(
                     source=QueueSources.PAY_API.value,
                     message_type=message_type,
                     payload=payload,
                     topic=current_app.config.get('ACCOUNT_MAILER_TOPIC')
-                )
+                ),
+                current_app
```

Comment:
no point in putting in current_app here as a param, you can get it from flask import

Reply:
`from flask import Flask, current_app`